src/node/rpc/test/frontend_test.cpp

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the Apache 2.0 License.
 #define DOCTEST_CONFIG_IMPLEMENT
+#define DOCTEST_CONFIG_NO_SHORT_MACRO_NAMES
```

Comment:
Why?

Reply:
Because the formatter puts the imported pbft file before it does the logger, and the test can't compile due to the LOG_INFO macros

Reply:
Does `#include <doctest/doctest.h>` (switching `" "` to `< >`) fix this?

Reply:
no, it also has a problem with `jsonrpc::MESSAGE` and `DOCTEST_MESSAGE`. It has to do with the fact that `consensus/pbft/pbftrequests.h` imports `jsonrpc.h` first and then we will import `doctest/doctest.h` afterwords causes them to clash.

Reply:
As discussed replacing `jsonrpc::MESSAGE` with the string `message` for now since it will change in upcoming PR's anyway.